packages/scraper-tools/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@apify/scraper-tools",
+    "version": "0.4.4",
+    "description": "Tools shared by Apify actor-scrapers.",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "author": {
+        "name": "Apify Technologies",
+        "email": "support@apify.com",
+        "url": "https://apify.com"
+    },
+    "contributors": [
+        "Marek Trunkat <marek@apify.com>",
+        "Ondra Urban <ondra@apify.com>"
+    ],
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/apify/apify-js"
+    },
+    "homepage": "https://github.com/apify/apify-js",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "build": "npm run clean && npm run compile && npm run copy",
+        "clean": "rimraf ./dist",
+        "compile": "tsc -p tsconfig.build.json",
+        "copy": "ts-node -T ../../scripts/copy.ts"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "dependencies": {
+        "ajv": "^8.8.2",
+        "content-type": "^1.0.4"
+    },
+    "peerDependencies": {
+        "apify": ">=2.x || >=2.1.1-beta.0"
+    },
+    "devDependencies": {
+        "@types/fs-extra": "^9.0.13",
+        "@types/puppeteer": "^5.4.4",
+        "apify": "file:../apify",
```

Comment:
you dont need dev deps on internal packages, they are available already due to TS paths mapping configuration

(just one think i noticed on the first look, will check this out locally before i give you more feedback)